### ISSUE TITLE
Unable to add or use SMB (bug)

### ISSUE THREAD

**Issue by Mobasheera** [zhanghai/MaterialFiles#813]
While connecting to my SMB server, it shows this error even though I had set it up for years now and suddenly it just does not work now. Other apps can access and add the SMB server but not Material Files. I hope this bug gets fixed asap as I heavily rely on it.
![image](https://user-images.githubusercontent.com/83943245/172990878-daecc1b8-8976-4540-b645-5e87e2cb4670.png)
![image](https://user-images.githubusercontent.com/83943245/172990897-c85eb5fc-df11-4220-939c-999cdf92af2e.png)

**Comment by mbnoimi** [comment]
The same bug still occurs in version 1.5.2 (31)
BTW, I'm able to access the same mount point using **X-plore File Manager** without any problem
![image](https://user-images.githubusercontent.com/284547/181936788-5ba1234b-e871-494e-a311-9fc5932b6d14.png)

**Comment by mbnoimi** [comment]
It seems this bug not solved since a while :(
https://github.com/zhanghai/MaterialFiles/issues/494

**Comment by Mobasheera** [comment]
yes that was exactly my problem, currently its working again so yeah, im closing this thread!

**Comment by Mobasheera** [comment]
.
